### PR TITLE
tweaking the password upgrading functionality

### PR DESCRIPTION
**Pull request by weaverryan**
Hi!

Minor tweaks to the password upgrading section as I read through it - it was already quite good.

### CODE REVIEW

File: security/password_migration.rst [hunk 1]
```diff
@@ -9,21 +9,20 @@ How to Migrate a Password Hash
...
 #. `Upgrade the Password`_
 #. Optionally, `Trigger Password Migration From a Custom Encoder`_
 
 Configure a new Encoder Using "migrate_from"
---------------------------------------------
+----------------------------------------------
 
-When configuring a new encoder, you can specify a list of legacy encoders by
-using the ``migrate_from`` option:
+When a better hashing algorithm becomes available, you should keep the existing
```
Comment (94noni) [inline security/password_migration.rst]: Safer/Stronger instead better?

File: security/password_migration.rst [hunk 2]
```diff
@@ -102,6 +102,13 @@ using the ``migrate_from`` option:
             ],
         ]);
 
+With this setup:
+
+* New users will be encoded with the new algorithm;
+* Whenever a user logs in whose password is still stored using the old algorithm,
+  Symfony will verify the password with the old algorithm and then re-encode
```
Comment (wouterj) [inline security/password_migration.rst]:
```suggestion
  Symfony will verify the password with the old algorithm and then rehash
```

Confusing Symfony terminology, but the password encoders are doing hashing. This is also consistent with earlier uses of "rehash(ing)" in the beginning of this chapter.

**Review by wouterj — APPROVED** [review]
Thanks, I agree with the improvements!

File: security/password_migration.rst (hunk 1 above)
Comment (javiereguiluz) [reply security/password_migration.rst]: I prefer to keep the generic word "better". I know it's generic, but that's why I preferred it in this case. For example, in the future we could have a new hash algorithm that it's as strong/safe as Argon2i, but consumes half of memory or requires half of storage. So, it wouldn't be safer/stronger, but better in other ways.

**Review by javiereguiluz — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Ryan.

File: security/password_migration.rst (hunk 2 above)
Comment (javiereguiluz) [reply security/password_migration.rst]: I made this change while merging.

**Pull request merged**